dsl/scripts/jobs.groovy

```diff
@@ -1059,6 +1060,20 @@ def jobs = [
             CHANGE_BRANCH: 'develop',
         ],
     ],
+    [
+        name: 'common/vegawallet-ui',
+        repo: 'vegawallet-ui',
+        useGithub: true,
+        jenkinsfile: 'Jenkinsfile',
```

Comment:
I do not think if this will work because there is no Jenkinsfile in the VegawalletUi repo:

```shell
➜  www cd vegawallet-ui      
➜  vegawallet-ui git:(main) git pull        
Already up to date.
➜  vegawallet-ui git:(main) ls -als | grep Jenkins
➜  vegawallet-ui git:(main) 
```